tee-worker/ts-tests/parachain-api/dist/src/index.js

```diff
@@ -1,17 +1,15 @@
 import "@polkadot/api/augment";
 import "@polkadot/types-augment";
 import { ApiPromise, Keyring, WsProvider } from "@polkadot/api";
-
-// Before we finalize the complete submission of the 'dist' files, we'll temporarily use the 'prepare-build' path here. Otherwise, CI can't find the files.
-import { identity } from "../../prepare-build/interfaces/definitions";
```

Comment:
This is the reason that CI failed.

According to @Verin1005 we should not touch it (or maybe manually reverted to the old version).

I had once similar problem where this file was changed automatically - I haven't figured out which step caused it

Reply:
That import is listed in the source for this lib; some of the test code is referencing the definitions directly (don't remember which exactly) so they should come from this package as a source of truth.

Problem was, while we relied on the `dist` folder being committed, the `dist/build` was gitignored, so without building the missing files, the committed build output is actually broken.

I want to sort this out properly along the course of #1878; in the meantime I just fixed the `.gitignore` so that if we commit `dist`, that means _all_ of `dist`.